packages/calcite-components/src/components/list-item/list-item.tsx

```diff
@@ -392,16 +392,23 @@ export class ListItem
     const { el, open, openable, parentListEl } = this;
     const dir = getElementDir(el);
 
+    const icon = openable
+      ? open
+        ? ICONS.open
+        : dir === "rtl"
+        ? ICONS.closedRTL
+        : ICONS.closedLTR
+      : ICONS.blank;
+
+    const iconNode = <calcite-icon icon={icon} key={icon} scale="s" />;
+
     return openable ? (
-      <td class={CSS.openContainer} key="open-container" onClick={this.toggleOpen}>
-        <calcite-icon
-          icon={open ? ICONS.open : dir === "rtl" ? ICONS.closedRTL : ICONS.closedLTR}
-          scale="s"
-        />
+      <td class={CSS.openContainer} key="open-container-openable" onClick={this.toggleOpen}>
```

Comment:
Do you need unique keys on the parent for the fix to work? I would think the icon's key would suffice as it would ensure it gets recreated.

Could you also add a note about this workaround?

Reply:
Nope, removed.